application/src/test/java/org/opentripplanner/apis/vectortiles/DebugStyleSpecTest.java

```diff
@@ -1,27 +1,29 @@
 package org.opentripplanner.apis.vectortiles;
 
+import static org.opentripplanner.framework.io.FileUtils.readFile;
+import static org.opentripplanner.framework.io.FileUtils.writeFile;
 import static org.opentripplanner.test.support.JsonAssertions.assertEqualJson;
+import static org.opentripplanner.test.support.JsonAssertions.isEqualJson;
 
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
+import java.io.File;
 import java.util.List;
 import org.junit.jupiter.api.Test;
 import org.opentripplanner.apis.vectortiles.model.TileSource.VectorSource;
 import org.opentripplanner.apis.vectortiles.model.VectorSourceLayer;
 import org.opentripplanner.framework.json.ObjectMappers;
 import org.opentripplanner.standalone.config.framework.json.JsonSupport;
-import org.opentripplanner.test.support.ResourceLoader;
 
 class DebugStyleSpecTest {
 
-  private final ResourceLoader RESOURCES = ResourceLoader.of(this);
+  private static final File STYLE_FILE = new File(
```

Comment:
By the way, in a different branch I needed something very similar and have written the following helper in ResourceLoader: https://github.com/leonardehrenfried/OpenTripPlanner/commit/ca59de72a15c86900c005e01fc3bccfafcee79d7

If you want, you can cherry-pick and modify it. Otherwise I will use it later.